lib/modules/platform/gitlab/types.ts

```diff
@@ -37,6 +37,38 @@ export interface GitLabMergeRequest {
   };
 }
 
+export interface GitlabCommit {
```

Comment:
this should be a zod schema instead of a simple interface.

we're currently refactor all interfaces to zod validating schema

Reply:
I moved GitlabCommit interface to zod schema